spec/models/user_spec.rb

```diff
@@ -1,5 +1,48 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let (:review) { FactoryGirl.create(:review) }
+  let (:user) { review.product.user}
+  it { is_expected.to have_secure_password }
+
+  describe ".validates" do
+    it { is_expected.to validate_presence_of(:email) }
+
+    it "must have a unique email" do
+      expect(User.new(name: "Nemo", username: "Nemo1", email: user.email, password: "123", password_confirmation: "123")).to_not be_valid
+    end
+
+    it "downcases email before checking validity" do
+      expect(User.new(name: "Nemo", username: "Nemo1", email: user.email.upcase, password: "123", password_confirmation: "123")).to_not be_valid
```

Comment:
The it  on line 15 states that `it 'downcases email before checking validity' do`, but the test block isn't checking that. It looks to me like it's checking a downcase validation. I'd either reword the 'it' block to decrease ambiguity or change the expect block. You could say `it "doesn't allow uppercase email" do` or something similar.